uploadNeuroDB/minc_insertion.pl

```diff
@@ -46,6 +46,10 @@
 my $NewScanner  = 1;           # This should be the default unless you are a 
                                # control freak
 my $xlog        = 0;           # default should be 0
+my $acquisitionProtocol;       # Specify the acquisition Protocol also bypasses the checks
+my $acquisitionProtocolID;     # acquisition Protocol id
+my @checks      = ();          # Initialise the array
+my $create_minc_pics    = 0;   # Default is 0, set the option to overide.
```

Comment:
@gluneau 
on line 63, can you please fix the typo that was already there (casic should be basic)!